8.0/root/opt/app-root/etc/scl_enable

```diff
@@ -0,0 +1,6 @@
+# IMPORTANT: Do not add more content to this file unless you know what you are
+#            doing. This file is sourced everytime the shell session is opened.
+#
+# This will make scl collection binaries work out of box.
+unset BASH_ENV PROMPT_COMMAND ENV
+source scl_source enable ${SCL_ENABLED} httpd24 $NODEJS_SCL
```

Comment:
This file can be removed as we do not have SCL variant for php 8.0.

Reply:
Addressed by https://github.com/sclorg/s2i-php-container/pull/347/commits/b2bfb6c3f2fe144f39784203075ac243a0216f72